src/app/components/ChartbeatAnalytics/utils/index.test.ts

```diff
@@ -226,21 +217,21 @@ describe('Chartbeat utilities', () => {
         service: 'korean',
         pageType: MEDIA_PAGE,
         description: 'should return expected section for live radio',
-        mediaPageType: 'On Demand Radio',
+        mediaPageType: 'Radio',
```

Comment:
Good question! Because there used to be a look up object, which mapped this media page type to a different one. I wanted to simplify the code, and remove the need to do this. 

https://github.com/bbc/simorgh/blob/latest/src/app/components/ChartbeatAnalytics/utils/index.ts#L135-L140

Reply:
Ah so we aren't interested in distinguishing between luve and OD radio?

Reply:
Looks like the data we send to Chartbeat doesn't distinguish between them... 

**Live Radio**
![image](https://github.com/bbc/simorgh/assets/58214768/f6dfa3ee-93d8-4255-b69b-34d681b9700f)

**On Demand Radio**
![image](https://github.com/bbc/simorgh/assets/58214768/3d7e5e7e-f2d9-4e12-b3a5-4dfc0bc293eb)


They can use the URLs to distinguish between them though...